### DISCUSSION TITLE
Open to contributing for the French translation of Tensorflow documentation

### DISCUSSION THREAD

**Discussion by Alexandre34** [tensorflow/docs-l10n#435] [Ideas]
Hi all,
I would like to contribute for the translation of the Tensorflow documentation in French.
Seeing the GitLocalize french branch, there seem to be a lot of missing documentation and no moderator.

I have some experience on Tensorflow and excellent English knowledge, so I would like to know how I can help, and especially, where to start first.

Best regards,
Alexandre

**Comment by leeyspaul** [comment]
I have the same question, similar to @Alexandre34 but for Spanish. I'm having the same questions but per this thread https://github.com/tensorflow/docs-l10n/discussions/418 it seems like an informal process thus far. 

I want to add the question of how one can be added as a moderator if we have the ability to review in a target language?

cc. @lamberta , tagging you for any insight if possible. :]

**Comment by lamberta** [comment]
Thanks for volunteering, @Alexandre34 !

Still figuring out what to do with French translations since, at this time, it's not a priority translation for us (meaning we have resources to support it). And, historically, there has not been much community support for French so it becomes difficult to get reviews and maintain. But, you're welcome to submit a translation pull request using GitLocalize and see if the community responds 😀
